lib/Checkout/Metadata/MetadataClient.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Checkout\Metadata;
+
+use Checkout\ApiClient;
+use Checkout\AuthorizationType;
+use Checkout\CheckoutApiException;
+use Checkout\CheckoutConfiguration;
+use Checkout\Client;
+use Checkout\Metadata\Card\CardMetadataRequest;
+
+class MetadataClient extends Client
+{
+    public function __construct(ApiClient $apiClient, CheckoutConfiguration $configuration)
+    {
+        parent::__construct($apiClient, $configuration, AuthorizationType::$oAuth);
```

Comment:
Shouldn't `AuthorizationType` be `secretKeyOrOAuth`?

Reply:
Yes you're right good catch